build_docs.pl

```diff
@@ -57,10 +57,11 @@ BEGIN
     'single',  'pdf',     'doc=s',           'out=s',  'toc', 'chunk=i',
     'open',    'skiplinkcheck', 'linkcheckonly', 'staging', 'procs=i',         'user=s', 'lang=s',
     'lenient', 'verbose', 'reload_template', 'resource=s@', 'asciidoctor', 'in_standard_docker',
+    'conf=s',
 ) || exit usage();
 check_args();
 
-our $Conf = LoadFile('conf.yaml');
+our $Conf = LoadFile(pick_conf());
```

Comment:
I think it'd be nice if we didn't rely on the conf file when building a single document but that is a topic for another time. And having these tests will help when we go to do that.